README.md

```diff
@@ -31,7 +31,7 @@ To create a connection, construct a connection URL by appending optional connect
 The default URL for Timestream is `jdbc:timestream`.
 
```

Comment:
On line 20 (just before `## Driver Class`), please add something like:
`- Instructions for using fully shaded JAR file with [DBVisualizer](markdown/setup/dbvisualizer-driver-setup.md)` so users know how to find the document